frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -258,6 +264,9 @@ export const HorizontalNav = React.memo((props: HorizontalNavProps) => {
   );
 
   const objReference = props.obj?.data ? referenceFor(props.obj.data) : '';
+  const [dynamicNavTabExtensions, navTabExtentionsResolved] = useResolvedExtensions<
+    DynamicHorizontalNavTab
+  >(DynamicIsHorizontalNavTab);
```

Comment:
looks like all the static extensions have been converted to use dynamic. we can remove the usage of the static plugin from this file and only keep dynamic. WDYT?

Reply:
We should keep this as `ceph-storage-plugin` and `metal3-plugin` packages are still using the static plugin.

Reply:
shouldn't we migrate those as well?

Reply:
I was not able to setup cluster storage so I leave it for now for the packages `ceph-storage-plugin` and `metal3-plugin`.